docs/pages/blog/mui-x-v5.3.0.md

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Introducing Row Grouping Feature'
+description: After more than a year of development on the DataGrid, the first premium feature is live.
+date: 2022-01-15T00:00:00.000Z
+authors:
+  ['oliviertassinari', 'm4theushw', 'flaviendelangle', 'DanailH', 'alexfauquette']
+card: true
+---
+
+<!-- <img src="/static/blog/2021/card.png" alt="" style="width: 100%; margin-bottom: 16px;" /> -->
+
+# A step to Premium plan: Row Grouping
+
+A lot of you were asking about it, and we are pleased to announce that the Premium plan is on the road.
+After an incredible year fully focused on improving the DataGrid, we are moving forward by launching the first feature of the Premium plan: the row grouping which is released in v5.3.0.
+
+Let me introduce this new feature.
+
+## Start navigating the data 🚢
+
+If you already use the pro plan, you may be familiar with the [tree data](/components/data-grid/group-pivot/#tree-data) which allows your users to navigate in the hierarchy by opening/closing children of a row.
+However, everything does not have a natural hierarchy, and users might like to modify it.
+Good news!
+It is now possible with row grouping.
+
+Let's play with the top 250 movies according to IMDb.
+There is not a clear hierarchy to organize movies.
+Should you group them by director, box office results or year of release?
+The answer depends on what the user wants to do.
+So let them be free to choose their own organization.
+With the row grouping, they can go in the option menu of the “director” column and click on “Group by director” to group all the rows with the same director.
+If they are not interested anymore by the director, they can simply click on “Stop grouping by director”.
```

Comment:
I believe https://www.quora.com/Is-it-grammatical-to-say-someone-is-interested-by-or-in-something

```suggestion
If they are not interested anymore in the director, they can simply click on “Stop grouping by director”.
```